### ISSUE TITLE
Use Drupal.org's API for queries/string_change.txt & rn_mention.txt

### ISSUE THREAD

**Issue by drumm** [xjm/drupal_core_release#2]
To get a term ID from a string:

https://www.drupal.org/api-d7/taxonomy_term.json?vocabulary=9&name=String%20change%20in%208.0.2&full=0

To query for issues:

https://www.drupal.org/api-d7/node.json?status=1&field_project=3060&field_issue_version=8.0.x-dev&field_issue_category=1&field_issue_status=7&taxonomy_vocabulary_9=36890

RestWS doesn't support OR/IN queries, so all combinations of field_issue_category & field_issue_status will have to be requested and results merged together.

**Comment by cilefen** [comment]
@drumm I was just checking this out and experienced intermittent 500 errors on queries like: https://www.drupal.org/api-d7/node.json?type=project_issue&field_project=3060&&field_issue_version=8.2.x-dev&field_issue_priority=300&field_issue_status=13

Also, is there a way to query multiple values, for example: &field_issue_status[]=13&field_issue_status[]=8?

**Comment by cilefen** [comment]
@drumm Duh, your comment on OR/IN went in one ear and out the other, sorry!

**Comment by drumm** [comment]
We do see the intermittent 500s on our end too. That seems to be the field cache warming up, we might lower the max number of entities returned in the future. Go ahead and retry, waiting 5-10s in between may increase success.

**Comment by drumm** [comment]
It turns out OR seems to be possible: https://www.drupal.org/node/2308939#comment-11371609

**Comment by cilefen** [comment]
Yes, that works: https://www.drupal.org/api-d7/node.json?type=project_issue&field_project=3060&&field_issue_version=8.2.x-dev&field_issue_priority=300&field_issue_status[value][]=13&field_issue_status[value][]=8&limit=10

**Comment by cilefen** [comment]
Or it _seems_ to work, in this case. There are comments in the issue you referenced on not-so-subtle surprises.
